src/org/zaproxy/zap/extension/pscanrulesBeta/ServletParameterPollutionScanner.java

```diff
@@ -75,20 +73,10 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 			// check for 'target' param
 			
 			for (Element formElement : formElements) {
-				boolean actionFound = false;
-				Attributes atts = formElement.getAttributes();
-				Iterator<Attribute> iter = atts.iterator();
-				
-				while (iter.hasNext()) {
-					Attribute att = iter.next();
-					if (att.getName().equalsIgnoreCase("action") && att.getValue().length() > 0) {
-						// action tag present (and with a value), so should be ok
-						actionFound = true;
-					}
-				}
-				
-				if (!actionFound) {
-				    Alert alert = new Alert(getPluginId(), Alert.RISK_MEDIUM, Alert.CONFIDENCE_LOW, 
+				boolean actionMissingOrEmpty = StringUtils.isEmpty(formElement.getAttributeValue("action"));
+
+				if (actionMissingOrEmpty) {
```

Comment:
I think you might need to use `isBlank()` to get null handling or `(formElement.getAttributeValue("action")!=null && !StringUtils.isEmpty(formElement.getAttributeValue("action")))`. (In the later case you may as well use an intermediate var for the String...)

```
public class Thing {

    public static void main(String[] args) {
	String thing=null;
	if (thing.isEmpty()) {
		System.out.println("IsEmpty");
	}
    }
}
```
Results in a NPE.
```
Exception in thread "main" java.lang.NullPointerException
        at Thing.main(Thing.java:6)
```